region/maridia/outer/Mt. Everest.json

```diff
@@ -1699,6 +1658,65 @@
       "notable": false,
       "requires": []
     },
+    {
+      "link": [4, 11],
+      "name": "Suitless Shinespark",
+      "notable": false,
+      "entranceCondition": {
+        "comeInWithSpark": {}
+      },
+      "requires": [
+        {"shinespark": {"frames": 127, "excessFrames": 28}}
+      ],
+      "devNote": "With enough energy, this goes all the way to 1, but with less it stops prematurely at 11."
+    },
+    {
+      "link": [5, 1],
+      "name": "Suitless Shinespark",
+      "notable": false,
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 90
+        }
+      },
+      "requires": [
+        "canSuitlessMaridia",
+        {"shinespark": {"frames": 61, "excessFrames": 10}}
+      ],
+      "devNote": "This could be done with Gravity to save Energy, but getting from 5 to 1 with Gravity and SpeedBooster is free."
+    },
+    {
+      "link": [5, 4],
+      "name": "Midair Shinespark",
+      "notable": false,
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 55
+        }
+      },
+      "requires": [
+        "canSuitlessMaridia",
```

Comment:
I think Gravity is missing here.

And there could be shinesparks which do not need to reach the grapple blocks first, in both directions.

Reply:
Good catch. 

I'll look to add those strats next I have time. I also should also add strats coming in shinecharging from the right with gravity, and sparking to the grapple blocks with a down grab from both sides (Probably just suitless sparks as it's easier to gravity jump up there if you have it).